src/test/scala/uk/gov/hmrc/test/ui/pages/mrc/PreviewSummaryPage.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.test.ui.pages.mrc
+
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.{FindBy, How, PageFactory}
+import uk.gov.hmrc.test.ui.pages.BasePage
+
+object PreviewSummaryPage extends BasePage {
+  @FindBy(how = How.XPATH, using = "//h1[@class='govuk-heading-l']") var lblHeader: WebElement        = _
+  @FindBy(how = How.XPATH, using = "//a[@class='print-this__print']") var linkPrint: WebElement       = _
+  @FindBy(how = How.XPATH, using = "//*[@id=\"pdf-preview\"]/div[2]/div[1]/h2") var tblCT: WebElement = _
+  @FindBy(how = How.XPATH, using = "//*[@id=\"pdf-preview\"]/div[2]/div[2]/h2") var tblET: WebElement = _
+
+  PageFactory.initElements(driver, this)
+
+  def verifyHeader(): String =
+    lblHeader.getText
+
+  def verifyLinkPrint(): String =
+    linkPrint.getText
+
+  def verifyTables(): Unit =
+    tblCT.isDisplayed
+  tblET.isDisplayed
```

Comment:
Do the above lines needs wrapping in {} ?

Reply:
Oopsy, updated it now